### PR TITLE
make chart

### PR DESCRIPTION
**Pull request by huuanh99**
## Related Tickets:
- [#44913](https://edu-redmine.sun-asterisk.vn/issues/44913)

## WHAT this PR does?
make order chart sale

## Notes Impacted Areas:
- N/A.

## Attachments:
Capture

### CODE REVIEW

**Comment by huuanh99** [comment]
ready

File: resources/views/admin/chart/orderchart.blade.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        <title>Order Chart</title>
+        <script src="{{ asset('js/admin/Chart.min.js') }}" charset="utf-8"></script>
+    </head>
+    <body>
+        <div>
+            {!! $chart->container() !!}
+        </div>
+        {!! $chart->script() !!}
+    </body>
+</html>
```
Comment (ghost) [inline resources/views/admin/chart/orderchart.blade.php]: thiếu dòng trắng cuối file

File: resources/views/admin/chart/orderchart.blade.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Order Chart</title>
```
Comment (ghost) [inline resources/views/admin/chart/orderchart.blade.php]: localization

File: resources/views/admin/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<div class="grid_10">
+  <div class="box round first grid">
+      <h2>{{ __('dashboard') }}</h2>
+      <div class="block">
+        {{ __('welcomeadmin') }}               
+      </div>
+  </div>
+</div>
+@endsection
+
```
Comment (ghost) [inline resources/views/admin/index.blade.php]:
```suggestion
```

File: resources/views/admin/layout.blade.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                    <div class="floatleft marginleft10">
+                        <ul class="inline-ul floatleft">
+                            <li>{{ __('hello') }} {{ Auth::user()->name }}</li>
+                            <li><form method="POST" action="{{ route('logout') }}">
+                                @csrf
+                                <x-dropdown-link :href="route('logout')"
+                                                onclick="event.preventDefault();
+                                                this.closest('form').submit();">
+                                        {{ __('Log Out') }}
+                                </x-dropdown-link>
```
Comment (ghost) [inline resources/views/admin/layout.blade.php]: indent loạn hết lên rùi nè

**Comment by huuanh99** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**